src/super_gradients/training/utils/callbacks/callbacks.py

```diff
@@ -856,6 +856,22 @@ def _infer_global_step(self, context: PhaseContext, is_train_loader: bool):
             return total_steps_in_done + train_loader_length + context.batch_idx
 
 
+@register_callback(Callbacks.SLIDING_WINDOW_INFERENCE)
+class ChangeToSWI(Callback):
+    """
+    Performing single-scale sliding window during inference at the end of training on the validation set.
+    """
+
+    def on_validation_loader_start(self, context: PhaseContext) -> None:
```

Comment:
Please check if average_best_models=True like we talked, and if so actiavte a newly added phase in average best model validation.

Reply:
Fixed on c79d0f2